clientv3/txn_test.go

```diff
@@ -98,4 +100,6 @@ func TestTxnPanics(t *testing.T) {
 			t.Errorf("#%d: did not panic, wanted panic %s", i, tt.err)
 		}
 	}
+
+	testutil.WaitPanic()
```

Comment:
You are right. I think we should call AfterTest and add panic into bad stuff. AfterTest is used to handle these slow shutdown things.